custom_components/tuya_local/devices/rotenso_ronix_heatpump.yaml

```diff
@@ -0,0 +1,309 @@
+name: Rotenso Roni X
+products:
+  - id: hw50w7qvxluhslkk
+    name: Rotenso Roni X
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: cold
+              value: cool
+            - dps_val: hot
+              value: heat
+            - dps_val: wet
+              value: dry
+            - dps_val: wind
+              value: fan_only
+            - dps_val: auto
+              value: heat_cool
+    - id: 2
+      type: integer
+      name: temperature
+      range:
+        min: 160
+        max: 880
+      mapping:
+        - scale: 10
+          step: 10
+    - id: 3
+      type: integer
+      name: current_temperature
+    - id: 4
+      type: string
+      name: mode
+      hidden: true
+    - id: 5
+      type: string
+      name: fan_mode
+      mapping:
```

Comment:
values should be standard values from https://github.com/home-assistant/core/blob/dev/homeassistant/components/climate/strings.json (under fan_mode) or extended values from https://github.com/make-all/tuya-local/blob/main/custom_components/tuya_local/translations/en.json lines 67-73, in which case `translation_key: aircon_extra` needs to be added at the entity level.